src/styles/main.scss

```
@@ -1,3 +1,56 @@
+$boxSize: 100px;
+$borderSize: 1px;
+$dayscount: 7;
+$days: ("mon": 0, "tue": 1, "wed": 2, "thu": 3, "fri": 4, "sat": 5, "sun": 6);
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  position: absolute;
+
+  width: $boxSize * $dayscount;
+
+  > div {
```

Додай це, але посортуй за правилами!
```suggestion

  width: $boxSize * $dayscount;
  transform: translate(-50%, -50%);

  > div {
```